examples/dt/nova/nova-three-cells/README.md

```diff
@@ -0,0 +1,185 @@
+# Deployed Topology - Nova/MultipleCells
+
+Deploys an environment with three cells instead of the standard two cells
+
+## Purpose
+
+Focused on leveraging multiple nodesets to deploy two distinct cells 
+(cell1 and cell2) in addition to cell0. Computes will be allocated to both 
+cells
+
+## Environment
+
+### Nodes
+
+| Role              | Machine Type | Count |
+| ----------------- | ------------ | ----- |
+| Compact OpenShift | vm           |   3   |
+| OpenStack Compute | vm           |   4   |
```

Comment:
Does this DT really deploys 4 EDPM nodes as computes? I only see edpm-compute-0 and edpm-compute-1 referenced in the NodeSets.

Reply:
Because this is completely virtualized its unfortunately not very clear based on purely this DT.  This DT works in conjunction with templates in CIFMW [1] that splits the total computes between each nodeset.   It also needs a scenario file similar to [2] to define the number of VMs needed (the one I'm using for testing this is define downstream).  Virtualized setups now have unique hostnames that are recorded during setup and that information is what [1] uses to allocate how the hosts are split.  With baremetal setups where I can define the hostnames in the jobs, I can specifically define how the EDPM Hosts are allocated per nodeset within the context of the DT.

[1] https://github.com/openstack-k8s-operators/ci-framework/pull/2423/files
[2] https://github.com/openstack-k8s-operators/ci-framework/blob/main/scenarios/reproducers/va-nfv-ovs-sriov.yml

Reply:
Yes that makes sense and you are correct the FW is overriding it to create the 4. Since that can't be clearly conveyed within just to context of the DT, I will update to 2.